orbit/pkg/scripts/scripts.go

```diff
@@ -103,7 +103,7 @@ func (r *Runner) runOne(script *fleet.HostScriptResult) (finalErr error) {
 		}()
 	}
 
-	ext := ".sh"
+	var ext string
 	if runtime.GOOS == "windows" {
 		ext = ".ps1"
 	}
```

Comment:
this makes me wonder if we could get rid of the extension altogether

Reply:
I was thinking the same thing. Windows might be picky about the file extension but I don't know